pkg/cri/resource-manager/policy/builtin/static-pools/stp-policy.go

```diff
@@ -135,13 +108,15 @@ func (stp *stp) Description() string {
 func (stp *stp) Start(add []cache.Container, del []cache.Container) error {
 	var err error
 
-	if stp.conf == nil {
-		return stpError("cannot start without any configuration")
+	if err := stp.nodeUpdater.start(); err != nil {
+		return err
 	}
 
-	err = stp.updateNode(*stp.conf)
-	if err != nil {
-		stp.Fatal("%v", err)
+	if stp.conf == nil {
+		err := stp.setConfig(cfg)
+		if err != nil {
+			return err
+		}
```

Comment:
Wondering about same as in [478](https://github.com/intel/cri-resource-manager/pull/478#discussion_r510990744).

Reply:
Fixed (in #478)